src/app/info/InfoPage.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import React, { useState, useEffect } from "react";
+import WikiItems from "@/app/info/WikiItems";
+
+import { Supabase } from "@/supabaseUtils";
+import { DbWikiRow } from "@/databaseUtils";
+import supabase from "@/supabaseClient";
+import Error from "next/error";
+
+interface QueryType {
+  data: DbWikiRow | null;
+  error: Error | null;
+}
+
+const getTopWikiRow = async (supabase: Supabase): Promise<QueryType> => {
+    const query = (await supabase.from("wiki").select("*").limit(1).single()) as QueryType;
+    return query;
+};
+
+const InfoPage: React.FC = () => {
+    const [topRow, setTopRow] = useState<DbWikiRow | null>(null); 
+    try {
+      useEffect(() => {
+        const fetchData: () => Promise<void> = async () => {
+            const temp: QueryType = await getTopWikiRow(supabase);
+            if (temp.error) {
+              throw temp.error;
+            } else if(temp.data) {
+              setTopRow(temp.data);
+            }
+        };
+        fetchData();
+      }, []);
+    } catch (e: any) {
+      return (
+      <>
+        <h3>Error Occured</h3>
+        <p>{e}</p>
+      </>
+      );
+    }
+
+    return (topRow && topRow.order !== -1) ? <WikiItems row={topRow}/> : <></>;
```

Comment:
order should never be -1 now I think, so this can be more concisely written
`return topRow && <wikiItems row={topRow}/>`

Reply:
removed the topRow condition as error is handled beforehand implying that data not be not null. However -1 is our default value for order, functions like an 'invalid' flag for a given wiki so that it wont be displayed. In the next ticket we filter out any -1 orders but since only one item is displayed in this ticket this is sufficient.